config/django_config/production.py

```
@@ -6,12 +6,15 @@
 
 from .base import *  # noqa
 
-DEBUG = env.bool("DJANGO_DEBUG", default=False)
-ALLOWED_HOSTS = env.list("ALLOWED_HOSTS", default=[])
+# DEBUG = env.bool("DJANGO_DEBUG", default=False)
+DEBUG = True
+ALLOWED_HOSTS = env.list("ALLOWED_HOSTS", default=["*"])
```

Ruff (0.8.2)

`env` may be undefined, or defined from star imports

(F405)